ext/quick-start-guide.html

```diff
@@ -24,57 +24,82 @@
 
     <span tabindex="-1" id="content-scroll-focus"></span>
 
-    <h1>Welcome to Yomitan!</h1>
+    <h1>Yomitan Quick Start Guide</h1>
+
+    <h2><strong>New to Yomitan?</strong></h2>
+    <div class="settings-group">
+        <a href="https://yomitan.wiki/" rel="noopener noreferrer" class="settings-item settings-item-button"><div class="settings-item-inner">
```

Comment:
I feel like this should link to the Get Started page on yomitan.wiki

Reply:
Actually I'm not sure about this link being here. I feel like the instructions of setting up Yomitan is pretty self contained here. Maybe link in the bottom for more info? I feel like going to the site is going to be less helpful for getting started than this quickstart guide.